cmd/osbuild-composer/testdata/test.toml

```diff
@@ -1,6 +1,9 @@
 [koji]
 allowed_domains = [ "osbuild.org" ]
 ca = "/etc/osbuild-composer/ca-crt.pem"
+enable_jwt = false
```

Comment:
Why did this change from `true` to `false`?

Reply:
because jwt + mtls (latter is true by default) is an invalid combo. I know it's just the config test but it seems weird tetsing an invalid config.

Reply:
ah, right!